docs/user/discover.asciidoc

```diff
@@ -178,7 +192,7 @@ you can use to build a structured query.
 Search the ecommerce data for documents where the country matches US:
 
 . Enter `g`, and then select *geoip.country_iso_code*.
-. Select *:* for equals some value and *US*, and then click *Refresh*.
+. Select *:* for equals some value and *US*, and then refresh the query.
```

Comment:
are we sure that the users knows what "refresh the query" means, maybe we could expain that by "refresh the query by clicking the *Refresh* button or hit the *Enter* key" ?